app/Models/Device.php

```diff
@@ -191,6 +193,11 @@ public function hasSnmpInfo(): bool
         return false; // no known snmpver
     }
 
+    public function snmpQuery(): SnmpQueryInterface
```

Comment:
This is not an API I want. Also it is forcing a specific SnmpQuery implementation.

Use SnmpQuery::device($device) to make queries against a device that is not the primary device.

Reply:
OK, thanks